apps/docs/content/guides/database/connecting-to-postgres.mdx

```diff
@@ -26,384 +20,121 @@ Supabase provides auto-updating Data APIs. These are the easiest way to get star
-
-<StepHikeCompact>
-
-  <StepHikeCompact.Step step={1}>
-
-    <StepHikeCompact.Details title="Install">
-
-    Install Drizzle and related dependencies.
-
-    </StepHikeCompact.Details>
-
-    <StepHikeCompact.Code>
-
-     ```shell
-     npm i drizzle-orm postgres
-     npm i -D drizzle-kit
-     ```
-
-    </StepHikeCompact.Code>
-
-  </StepHikeCompact.Step>
-
-  <StepHikeCompact.Step step={2}>
-
-    <StepHikeCompact.Details title="Create your models">
-
-    Create a `schema.ts` file and define your models.
-
-    </StepHikeCompact.Details>
-
-    <StepHikeCompact.Code>
-
-    ```ts schema.ts
-    import { pgTable, serial, text, varchar } from "drizzle-orm/pg-core";
-
-    export const users = pgTable('users', {
-      id: serial('id').primaryKey(),
-      fullName: text('full_name'),
-      phone: varchar('phone', { length: 256 }),
-    });
-    ```
-
-    </StepHikeCompact.Code>
-
-  </StepHikeCompact.Step>
-
-  <StepHikeCompact.Step step={3}>
-
-    <StepHikeCompact.Details title="Connect">
-
-    Connect to your database using the Connection Pooler.
-
-    In your [`Database Settings`](https://supabase.com/dashboard/project/_/settings/database), make sure `Use connection pooler` is checked, then copy the URI and save it as the `DATABASE_URL` environment variable. Remember to replace the password placeholder with your actual database password.
-
-    </StepHikeCompact.Details>
-
-    <StepHikeCompact.Code>
-
-    ```ts db.ts
-    import 'dotenv/config'
-
-    import { drizzle } from 'drizzle-orm/postgres-js'
-    import postgres from 'postgres'
-
-    const connectionString = process.env.DATABASE_URL
-
-    // Disable prefetch as it is not supported for "Transaction" pool mode
-    export const client = postgres(connectionString, { prepare: false })
-    export const db = drizzle(client);
-    ```
-
-    </StepHikeCompact.Code>
-
-  </StepHikeCompact.Step>
-
-</StepHikeCompact>
-
-### Connecting with pgAdmin
-
-[`pgAdmin`](https://www.pgadmin.org/) is a GUI tool for managing Postgres databases. You can use it to connect to your database via SSL:
-
-<StepHikeCompact>
-
-  <StepHikeCompact.Step step={1}>
-
-    <StepHikeCompact.Details title="Register">
-
-    Register a new Postgres server.
-
-    </StepHikeCompact.Details>
-
-    <StepHikeCompact.Code>
+### Supavisor session mode (port 5432):
 
-    <Image
-      alt="Register a new postgres server."
-      src={{
-          dark: '/docs/img/guides/database/connecting-to-postgres/pgadmin/register-server-pgAdmin.png?v=2',
-          light:
-            '/docs/img/guides/database/connecting-to-postgres/pgadmin/register-server-pgAdmin--light.png',
-        }}
-      />
+Supabase's Supavisor pooler in Session mode works just like a direct connection but with default IPv4 support. It serves as an IPv4 proxy for developers who prefer direct connection strings but are in IPv4-only environments or lack the IPv4 add-on.
```

Comment:
maybe we need to mention something like this:

In session mode, on each incoming connection, Supavisor creates a direct connection to the database, keeps it open until the client disconnects, and does not share it with other connections. Once the client disconnects, the connection is closed. This approach may differ from that of other database poolers